cloudflared/config.yaml

```diff
@@ -7,6 +7,7 @@ description: "Use a Cloudflare Tunnel to remotely
 url: "https://github.com/brenner-tobias/addon-cloudflared/"
 codenotary: dev@brenner.tech
 init: false
+webui: http://[HOST]:[PORT:36500]/metrics
```

Comment:
![image](https://github.com/user-attachments/assets/c1ea1960-fab0-4184-8e8e-7d1d3c865385)

If your intention was to make a button show up here, I think it's missing ingress configuration too.

```yaml
ingress: true
ingress_port: 36500
ingress_stream: true # if websockets are needed, not sure
```

Reply:
Nevermind, just saw https://github.com/brenner-tobias/addon-cloudflared/pull/730.

And I agree, I don't think the button should be added indeed for such interface.

BTW it worked fine:

![image](https://github.com/user-attachments/assets/56c65ee5-81f0-4e31-b83e-0e4c21efa04e)

Reply:
Thanks, I think we are good with the current implementation, let me know if you have some suggestions for improvements.